app/views/posts/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<section class="post-container">
+  <%= form_with(model: @post, :html => {:class => 'new_post'}, url: user_posts_path, method: :post, local: true) do |form| %>
+    <div class="form-group">
+      <%= form.label :title, class: 'label' %><br>
+      <%= form.text_field :title, class: 'field' %>
+    </div>
+
+    <div class="form-group">
+      <%= form.label :text, class: 'label' %><br>
+      <%= form.text_area :text, class: 'field' %>
+    </div>
+
+    <%= form.submit "Add Post", class: 'submit' %>
+
+  <% end %>
+</section>
```

Comment:
- [ ] I can't access this form. No navigation links to the form. All I can access is the root_path page:

![image](https://user-images.githubusercontent.com/51296741/156903389-7e0f7326-64fa-4c5f-9d2f-37bd94404c6b.png)
It will be nice to fix this.